src/js/components/AppListFilterComponent.jsx

```diff
@@ -71,13 +76,19 @@ var AppListFilterComponent = React.createClass({
   },
 
   blurInputGroup: function () {
-    console.log(this.state);
     this.setState({
       focused: false,
       activated: this.state.filterText !== ""
     });
   },
 
+  handleKeyDown: function (event) {
+    if (event.key === "Escape") {
```

Comment:
Neat! 

Weird though, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility) `event.key` is not supposed to be supported by Opera/Safari although I just tried it and it works. 
Maybe it's some React's synthetic event magic?